src/main/java/uk/gov/companieshouse/exemptions/controller/ControllerExceptionHandler.java

```diff
@@ -25,22 +26,23 @@ public class ControllerExceptionHandler {
 
     @ExceptionHandler(NotFoundException.class)
     public ResponseEntity<Void> handleNotFound(NotFoundException ex) {
-        LOGGER.error("Unable to locate company exemptions", ex);
+        LOGGER.error(ex.getMessage(), DataMapHolder.getLogMap());
         return ResponseEntity
                 .status(HttpStatus.NOT_FOUND)
                 .build();
     }
 
     @ExceptionHandler(ConflictException.class)
-    public ResponseEntity<Void> handleConflict() {
+    public ResponseEntity<Void> handleConflict(Exception ex) {
+        LOGGER.error(ex.getMessage(), DataMapHolder.getLogMap());
```

Comment:
The exception should be passed too when logging at error so the data map includes the stack trace, ```logger.error("message", ex, DataMapHolder.getLogMap());```